examples/megacli.cpp

```diff
@@ -3316,6 +3316,40 @@ void exec_cp(autocomplete::ACState& s)
 
 void exec_du(autocomplete::ACState& s)
 {
+    if (s.extractflag("-all"))
+    {
+        bool breakdown = s.extractflag("-breakdown");
+        struct counters { m_off_t filecount = 0, filebytes = 0, fileversioncount = 0, fileversionbytes = 0, foldercount = 0; };
+        map<Node*, counters> values;
+        for (auto& n : client->nodes)
```

Comment:
nodePair instead of n? can it be const?

Reply:
just following the existing style ;-)

Reply:
Single letter names? This practise was explicitly criticized by Pablo and I agree. We should have good, explanatory names.